fe/fe-core/src/main/java/org/apache/doris/statistics/AnalysisTaskInfoBuilder.java

```diff
@@ -109,7 +127,29 @@ public AnalysisTaskInfoBuilder setScheduleType(ScheduleType scheduleType) {
     }
 
     public AnalysisTaskInfo build() {
-        return new AnalysisTaskInfo(jobId, taskId, catalogName, dbName, tblName, colName,
-                indexId, jobType, analysisMethod, analysisType, message, lastExecTimeInMs, state, scheduleType);
+        return new AnalysisTaskInfo(jobId, taskId, catalogName, dbName, tblName,
+                colName, indexId, jobType, analysisMethod, analysisType, samplePercent,
+                sampleRows, maxBucketNum, message, lastExecTimeInMs, state, scheduleType);
+    }
+
+    public AnalysisTaskInfoBuilder deepCopy() {
```

Comment:
Since the fields of this class is just some enum and basic type, I think name it as `copy` is enough